packages/amplication-client/src/Components/EntitySelectField.tsx

```diff
@@ -11,27 +11,37 @@ type TEntities = {
   ];
 };
 
+type RelatedEntityOption = {
+  value: string;
+  label: string;
+};
+
 type Props = Omit<SelectFieldProps, "options"> & {
   resourceId: string;
 };
 
 const EntitySelectField = ({ resourceId, ...props }: Props) => {
+  const [relatedEntityListOptions, setRelatedEntityListOptions] =
+    React.useState<RelatedEntityOption[]>([]);
   const { data: entityList } = useQuery<TEntities>(GET_ENTITIES, {
     variables: {
       resourceId: resourceId,
     },
+    fetchPolicy: "no-cache",
```

Comment:
Is this on purpose? We shouldn't bypass cache apart for specific scenarios such as queries that return one time tokens or similar.

Reply:
Yes. This is on purpose. Without it, it just doesn't work

Reply:
this is probably due to same  requests that has been done before this component got rendered. this is bad performance and practice. this is the reason she need to cancel previous cache

Reply:
can we force the cache for this instead of never caching?

Reply:
you don't manage cache in two places. you do once source of truth